flaskbb/tokens/__init__.py

```diff
@@ -8,3 +8,6 @@
 """
 
 from .serializer import FlaskBBTokenSerializer
+
+
```

Comment:
Inconsistent spacing between imports and `__all__` -- do we want 1 or 2 newlines? Flake8 seems fine with either.